src/jit/flowgraph.cpp

```diff
@@ -9000,7 +9000,7 @@ void Compiler::fgSimpleLowering()
                         add->gtNext  = tree;
                         tree->gtPrev = add;
 #else
-                        range.InsertAfter(arr, con, add);
+                            range.InsertAfter(arr, con, add);
```

Comment:
I share your pain ;-)

---
In reply to: [77275413](https://github.com/dotnet/coreclr/pull/7027#discussion_r77275413) [](ancestors = 77275413)